target/linux/mvebu/image/cortexa9.mk

```diff
@@ -112,6 +118,7 @@ TARGET_DEVICES += linksys_wrt1900acs
 define Device/linksys_wrt1900ac-v1
   $(call Device/linksys)
   $(Device/dsa-migration)
```

Comment:
dsa-migration can obviously be dropped now, as it's the same two variables you reset with the new include.